docs/swagger-ui/src/main/scala/sttp/tapir/swagger/SwaggerUI.scala

```diff
@@ -75,9 +75,9 @@ object SwaggerUI {
     )
 
     if (options.pathPrefix == Nil)
-      List(yamlEndpoint, oauth2Endpoint, swaggerInitializerJsEndpoint, resourcesEndpoint)
+      List(yamlEndpoint, swaggerInitializerJsEndpoint, resourcesEndpoint, oauth2Endpoint)
     else
-      List(yamlEndpoint, redirectToSlashEndpoint, oauth2Endpoint, swaggerInitializerJsEndpoint, resourcesEndpoint)
```

Comment:
ha I'm suprised that this is all :) isn't there an infinite redirect loop at :49?

Reply:
At first attempt it will got to resources endpoint -> then if there is none resources specified it will go to redirect at :49 ->  which should then point back to resources endpoint  under correct path. :) So I think it suits our needs. 

Previously it goes to infinite indeed. 

I have checked this solution for default options and for pathPrefix = List("otherPath").

Where do you think such situation might still occur?

Reply:
Alternatively I could read resources in similar way as it is done for swaggerInitializer. Then endpoint  should be before resourcesEndpoint .:)

Reply:
From my perspective if we don't manipulate on content it is better to use approach I've proposed

Reply:
No, if this works that's completely fine, though I still don't understand how does this work ;) We're doing a redirect from `/docs/oauth2-redirect.html?xyz` to `/docs/oauth2-redirect.html?xyz` - what's the point? :)

Reply:
I've removed this redirect endpoint and checked that this is no longer needed  if it ever was.

Reply:
Ha interesting :) well the less code, the better :)